public/app/plugins/datasource/prometheus/querycache/QueryCache.test.ts

```diff
@@ -5,10 +5,21 @@ import { DataFrame, DataQueryRequest, DateTime, dateTime, TimeRange } from '@gra
 import { QueryEditorMode } from '../querybuilder/shared/types';
 import { PromQuery } from '../types';
 
-import { getTargSig, QueryCache } from './QueryCache';
+import { DatasourceProfileData, QueryCache } from './QueryCache';
 import { IncrementalStorageDataFrameScenarios } from './QueryCacheTestData';
 
-const mockRequest = (request?: Partial<DataQueryRequest<PromQuery>>): DataQueryRequest<PromQuery> => {
+// Will not interpolate vars!
+const interpolateStringTest = (query: PromQuery) => {
```

Comment:
do we need a function for that? Seems like it is just an attribute access

Reply:
So this is just a mock function, since the calling context of the queryCache needs to provide an implementation for interpolating variables, and in the actual datasource we have the interpolateString function, but to test the queryCache we don't really need to test variable interpolation which is tested in other places, so this just returns the query. We could potentially do a better job of mocking the datasource so the interpolate was available to us, but I didn't think that would provide us additional coverage, so I just mocked it out for now.